### ISSUE TITLE
Issue page : need button to submit on mobile

### ISSUE THREAD

**Issue by magsout** [webcompat/webcompat.com#328]
Search does not work on mobile, I think it is necessary to add a search button. Someone had already mentioned, and in addition to accessibility is better.
